sys/invitation.js

```diff
@@ -249,6 +251,19 @@ class FWInvitation {
       intercomm.bpublish(gid, obj.r, config.serviceType);
     }
 
+    const cmd = "awk '{print $1}' /proc/uptime";
```

Comment:
You can use async version of exec.

https://www.npmjs.com/package/child-process-promise